CMakeLists.txt

```
@@ -141,6 +142,13 @@ set(CMD_MAKE make)
 
 include(ExternalProject)
 
+if(USE_JEMALLOC)
+	if(USE_ASAN)
+		message(WARNING "Jemalloc and ASAN do not play well together")
```

Non blocking, what about:
```suggestion
		message(WARNING "Jemalloc and ASAN are known to have issues when combined")
```
?

---

Better ahah